docs/UserGuide.md

```diff
@@ -13,151 +27,211 @@ PeopleSoft is a desktop app for **calculating the salary for shift-based contrac
 
-`find Nicole` finds all entries with a person named ‘Nicole’
+`find Nicole` finds all the employees named ‘Nicole’
 
-`find * Hardware` finds all entries tagged with ‘Hardware’
+`find * Hardware` finds all employees tagged with ‘Hardware’
 
-`find Nicole Hardware Display` finds all entries with a person named ‘Nicole’, tagged with ‘Hardware’ AND ‘Display’ BOTH
+`find Nicole Hardware Display` finds all employees named ‘Nicole’, tagged with BOTH 'Hardware' AND 'Display'
 
 ### List all persons : `list`
-Lists all the employees in the company
+Lists all the employees in the company.
 
 Format: `list`
 
 Example: `list` shows all the employees in the company
 
 ### Clear all entries : `clear`
 
-Removes all the employees’ information in the company from the app. Useful for clearing out sample data. WARNING: You cannot recover the data afterwards.
+Removes all the employees’ information in the company from the app. Useful for clearing out sample data.<br>
+❗️Caution: You cannot recover the data afterwards.
 
-[coming in v1.2] Requests for confirmation from the user before clearing.
+[coming in v1.3] Requests for confirmation from the user before clearing. // ian's idea: get them to type 'yes clear' to confirm
 
 Format: `clear`
 
-Example: `clear` removes all the employees from the app.
+Example: `clear` removes all the employees from the app // ian: does it remove all jobs from the app too?
 
-### Exit the program : `exit`
-Exits the program immediately.
-[coming in v1.3] Requests for confirmation from the user before exiting.
+--------------------------------------------------------------------------------------------------------------------
 
-Format: `exit`
+## Job-related commands
 
-### Save the data
-PeopleSoft saves the data to the hard disk automatically after any command that changes the data. There is no need to save manually.
+### Add a job : `job`
 
-### Edit the data externally
-PeopleSoft data is saved as a JSON file [JAR file location]/data/peoplesoft.json. Advanced users are welcome to update data directly by editing that data file.
+Adds a new job to the system with the given attributes. `RATE` refers to how much the employee is paid per hour. `DURATION` refers to how long the job took.
 
-❗️Caution: If your changes to the data file makes its format invalid, PeopleSoft will discard all data and start with an empty data file at the next run.
+Format: `job JOBID n/NAME r/RATE d/DURATION`
+
+Example: `job 2 n/Fix HDB Lock r/40 d/1` creates a job with id 2, where the employees worked for 1 hour and are paid an hourly rate of $40 to fix a HDB lock
+
+### List all jobs : `joblist`
+
+Lists all the jobs you have created, including jobs that have been paid for and those that have not been paid for yet.
+
+Format: `joblist`
+
+Example: `joblist` lists all the jobs you have created
+
+### Delete a job : `jobdelete`
+
+Deletes the job that was referred to by the index.<br>
```

Comment:
is this line break necessary? why not just put in two lines (to denote an actual line break)

Reply:
all the other line breaks below too